Data/Scripts/System/Commands/Player/Skills.cs

```diff
@@ -43,7 +43,7 @@ public NewSkillsGump( Mobile from, int origin ) : base( 50, 50 )
 text = text + "Alchemy - This will need a mortar, pestle, empty bottle, and some reagents. A potion keg can hold many bottles of potions. Double clicking the mortar and pestle will start you on your way.<BR><BR>";
 text = text + "Anatomy - This will increase as you simply fight and heal with bandages. It allows for better damage and extra healing.<BR><BR>";
 text = text + "Arms Lore - This allows some crafters to make better weapons and armor. You may also find armor or weapons that are unidentifed. using this skill on them may identify them. You can identify weapons and armor to practice this skill, but they can only each be identified once. If you find some decorative armor or weapons, you can sometimes tell what you can get for it, and from whom. Again, these are decorative items and not something you would be able to use. If you determine that something like this does have value, give the item to the merchant you determined and they will give you some gold for it. Just hand it to them or keep it to decorate your home. If you master your mercantile skill, you will get double the price for rare items like these. If you want to have a chance for better weapon damage, this skill can help that as well.<BR><BR>";
-text = text + "Begging - This may get you a couple of gold if you use this skill on one of the many townsfolk. You can also target yourself for begging which will change your demeanor to 'begging'. While using this demeanor, townsfolk may do some services cheaper for you such as repairs, identification, and magic item charges. They will also be willing to buy your items and relics at a higher price. If you are really skilled at begging, you could convince opponents to stopping their attacks on you. Every time you are successful with this skill, you will lose a bit of karma. You will also lose fame if you cowardly convince one from attacking you.<BR><BR>";
+text = text + "Begging - This may get you a couple of gold if you use this skill on one of the many townsfolk. You can also target yourself for begging which will change your demeanor to 'begging'. While using this demeanor, townsfolk may do some services cheaper for you such as repairs, identification, and magic item charges. They will also be willing to buy your items and relics at a higher price. If you are really skilled at begging, you could convince opponents to stop their attacks on you. Every time you are successful with this skill, you will lose a bit of karma. You will also lose fame if you use this skill to convince an opponent from attacking you (this is considered a cowardly act.)<BR><BR>";
```

Comment:
What about something like:

If you are really skilled at begging, you could convince opponents to stop their attacks on you, but you will lose some fame for being cowardly. Every time you are successful with this skill, you will also lose a bit of karma.

Reply:
> What about something like:
> 
> If you are really skilled at begging, you could convince opponents to stop their attacks on you, but you will lose some fame for being cowardly. Every time you are successful with this skill, you will also lose a bit of karma.

I like this more actually!

Reply:
Made another commit replacing the text with yours! This should be all set now but I'll leave it for you to review. Thanks again!